ta/Makefile

```diff
@@ -4,25 +4,4 @@ CPPFLAGS += -DCFG_TEE_TA_LOG_LEVEL=$(CFG_TEE_TA_LOG_LEVEL)
 # The UUID for the Trusted Application
 BINARY=8aaaf200-2450-11e4-abe2-0002a5d5c51b
 
--include $(TA_DEV_KIT_DIR)/mk/ta_dev_kit.mk
```

Comment:
Why are you removing non-Android stuff in an "Add Android.mk" patch? Follow up, I remember the discussion about the targets below. How come you want to removed them now?